archetypes/Exchange/index.tsx

```diff
@@ -100,7 +109,7 @@ export default styled((({ onClose, className }) => {
             <Close onClick={onClose} className="close" />
             <Title>
                 <NetworkHintContainer>
-                    New Commit
+                    Open a Trade
```

Comment:
according to spec, this is supposed to change based on the selected commit type:
> ...
> On the Commit modal’s “Burn” tab: update h2 to “Close a Trade”
> On the Commit modal’s “Flip” tab: update h2 to “Reverse a Trade”
> ...